crates/cheatnet/src/forking/cache.rs

```diff
@@ -21,6 +22,7 @@ struct ForkCacheContent {
     class_hash_at: HashMap<String, String>,
     compiled_contract_class: HashMap<String, String>,
     compiled_class_hash: HashMap<String, String>,
+    block_info: Option<CheatnetBlockInfo>,
```

Comment:
Why this is optional?

Reply:
Because it is cached when the `fork_state_reader.get_block_info` is called for the first time